Services/Mail/classes/class.ilMail.php

```diff
@@ -659,13 +668,13 @@ protected function distributeMail(
                 $sentMailId
             );
         }
-        $this->setAutoResponderStatus(false);
-        if ($this->auto_responder_data) {
-            foreach ($this->auto_responder_data as $usr_id => $mail_options) {
+        $mail = $this->disableAutoResponder();
+        if ($mail->auto_responder_data) {
+            foreach ($mail->auto_responder_data as $usr_id => $mail_options) {
                 $tmpmail = new ilFormatMail($usr_id);
                 $tmpmail->setSaveInSentbox(false);
                 $tmpmail->sendMail(
-                    ($this->loginByUsrIdCallable)($this->user_id),
+                    ($mail->loginByUsrIdCallable)($mail->user_id),
                     '',
                     '',
                     $mail_options->getAbsenceAutoResponderSubject(),
```

Comment:
Das würde ich noch anders machen. Da wir uns gerade *in* der Instanz bewegen, würde ich hier einfach die Property auf `false` setzen, ohne über den `Mutator` zu gehen. D.h. einfach hier überall `$this` benutzen.